tests/test_client.py

```diff
@@ -0,0 +1,167 @@
+import pytest
+import testinfra
+import time
+import subprocess
+
+
+@pytest.fixture(scope='session')
```

Comment:
should this still be scoped to the session? I think each iteration should get a new client.

Reply:
The problem I can into was that this fixture requires other `session` fixtures and pytest complained. So I have to make this a session or de-session all the rest of the fixtures. I am happy to go either way.

Reply:
I think I'd prefer this not being `session` and thus the others to follow, as `session` implies caching of fixtures, and we really want a new container for every test.

@ekohl what do you think?